keras_cv/layers/preprocessing/random_saturation.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import numpy as np
+import tensorflow as tf
+
+from keras_cv.utils import preprocessing
+
+
+class RandomSaturation(tf.keras.__internal__.layers.BaseImageAugmentationLayer):
+    """Randomly performs the saturation adjustment on given images.
+
+    This layer will randomly increase/reduce the saturation for the input RGB
+    images. At inference time, the output will be identical to the input.
+    Call the layer with `training=True` to adjust the brightness of the input.
+
+    Note that different saturation adjustment factors will be apply to each the
```

Comment:
I wonder if we need to include:

"    Note that different saturation adjustment factors will be apply to each the images in the batch" -> 

This is true for all of our layers so it can probably just be assumed.

Reply:
Ack. Removed. (This was noticed on the core keras KPLs.)